src/clarity-angular/checkboxes/checkbox.ts

```diff
@@ -82,14 +86,17 @@ export class Checkbox implements ControlValueAccessor {
     public change = new EventEmitter<boolean>(false);
 
     public toggle() {
-        this._indeterminate = false;
```

Comment:
Oh, yeah, I misread that.

I was thinking of a case where checked stays the same so we wouldn't set indeterminate to false, but that a `toggle()` method, so obviously it can't happen.

👍